.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+* [ ] Have you followed our guidelines in CONTRIBUTING.md? 
+* [ ] Have you self-reviewed your code and corrected any misspellings? 
```

Comment:
This one is funny given the many spellings of "indices" in the original code

Reply:
Haha, I know. Also given that I had several misspellings in this original PR. Would you remove it?

Reply:
I think this is a great policy to have, actually. I'm 100% on board with requiring this.